lib/statsd/instrument/backends/udp_backend.rb

```diff
@@ -84,6 +84,11 @@ def write_packet(command)
       synchronize do
         socket.send(command, 0) > 0
       end
+    rescue ThreadError => e
```

Comment:
It shouldn't, and here's why: if we re-raise in the case of a container sending a stat on exit, it will cause it to shut down in an unclean way, which makes deploys fail.